repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -581,7 +687,7 @@ class Interpreter(prompt0: Ref[String],
       Interpreter.this.compiler.search(target)
     }
     def compiler = Interpreter.this.compiler.compiler
-    def newCompiler() = init()
+    def newCompiler() = null
```

Comment:
Not sure what you are doing here, but returning `null` is probably not the right thing to do